sdk/ml/azure-ai-ml/tests/test_utilities/utils.py

```diff
@@ -273,12 +275,22 @@ def delete_file_if_exists(file_path: str):
         os.remove(file_path)
 
 
+@contextmanager
+def skip_sleep_in_lro_polling(work_only_if_not_live=True):
+    if work_only_if_not_live and is_live():
+        yield
+        return
+
+    from azure.core.polling.base_polling import LROBasePolling
+    with patch.object(LROBasePolling, "_sleep", return_value=None):
```

Comment:
one risk : `_sleep` is a private method, so we have to be wary of its stability. Are there any other public methods we we can use?

Reply:
I have mocked RequestsTransport.sleep instead. Let's see if it works well